redwood-protocol-guest/src/commonMain/kotlin/app/cash/redwood/protocol/guest/ProtocolWidgetSystemFactory.kt

```diff
@@ -15,12 +15,23 @@
  */
 package app.cash.redwood.protocol.guest
 
+import app.cash.redwood.Modifier
+import app.cash.redwood.RedwoodCodegenApi
+import app.cash.redwood.protocol.ModifierTag
 import app.cash.redwood.widget.WidgetSystem
+import kotlinx.serialization.KSerializer
 
 public interface ProtocolWidgetSystemFactory {
```

Comment:
With these two new APIs I don’t think the `Factory` name fit as well as it did previously.